content/kubernetes/re-clusters/delete_custom_resources.md

```diff
@@ -0,0 +1,74 @@
+---
+Title: Delete custom resources
+linkTitle: Delete custom resources
+description: This article explains how to delete Redis Enterprise clusters and Redis Enterprise databases from your Kubernetes environment.
+weight: 90
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [
+    /kubernetes/re-clusters/delete_custom_resources.md,
+    /kubernetes/re-clusters/delete_custom_resources/,
+]
+---
+
+## Delete a database
+
+To delete a database managed by the Redis Enterprise Kubernetes operator, run `kubectl delete redb <your-db-name>` from your K8s cluster.
+
+When you delete a database, your data and the REDB custom resource file are also deleted.
+
+### Troubleshoot REDB deletion
+
+After a deletion request, the operator attaches a finalizer to the Redis Enterprise database (REDB) object. This makes sure the database is deleted before the REDB custom resource is removed from the K8s cluster.
+
+If the finalizer isn't removed automatically by the operator, you won't be able to delete your REDB resource.
+
+If this happens, you can remove the finalizer manually with the following command:
```

Comment:
This is probably worthy of a note of caution, since if the finalizer hasn't been removed, there's no guarantee that the underlying database has been deleted, which might cause resource leaks and may require manual intervention.